rasa_addons/core/policies/disambiguation.py

```diff
@@ -36,7 +36,7 @@ def __init__(
         self.fallback_default_confidence = 0.30
         self.disambiguation_action = "action_botfront_disambiguation"
         self.disambiguation_followup_action = "action_botfront_disambiguation_followup"
-        self.fallback_action = "action_botfront_fallback" # returns utter_fallback
+        self.fallback_action = "action_botfront_fallback"  # returns utter_default
```

Comment:
humm, I'm not sure to understand why it would change to returning utter_fallback

Reply:
Oh, because I changed it in actions/action_botfront_disambiguation.py too :D

Reply:
This will break existing projects using utter_fallback!

Reply:
Good point, and I guess that was your comment too, Guillaume. My impression was that utter_fallback could cause the most potential confusion for newcomers. But this is definitely an unnecessary breaking change. I'll revert this.